crates/vm/Cargo.toml

```diff
@@ -35,6 +35,8 @@ serde.workspace = true
 bincode = "1"
 
 ethereum-types.workspace = true
+rayon = "1.5"
```

Comment:
hmm, do we want to add a new dependency for this? Can't we use tokio? @jrchatruc wdyt?

Reply:
Rayon actually uses OS threads instead of light threads like tokio.
We could try and spawn some tasks that compute the tx senders but this is a CPU heavy task and might block Tokio's Scheduler (maybe we can use spawn_blocking). Rayon seems like an actual better fit for this task.

We **could** implement a thread-pool to avoid adding Rayon as a dependency (this is what essentialy Rayon abstracts away), but we might end up with a poor implementation, not as fast as this one.

May I suggest a middle-ground? Maybe we could cherry-pick rayon's features and come up with a smaller dependency, but I haven't researched if it is possible.